src/convenience.rs

```diff
@@ -0,0 +1,152 @@
+use std::collections::HashMap;
```

Comment:
I don't really see why this needs to be in prisma-query... And the naming of the file should be thought once more...